http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeV2ServiceInterp.scala

```diff
@@ -82,13 +82,7 @@ class RuntimeV2ServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
         .raiseUnless(hasPermission)(ForbiddenError(userInfo.userEmail))
 
       leoAuth <- samDAO.getLeoAuthToken
-      relayNamespaceOpt <- wsmDao.getRelayNamespace(workspaceId, req.region, leoAuth)
-      relayNamespace <- F.fromOption(
-        relayNamespaceOpt,
-        BadRequestException(s"Workspace ${workspaceId} doesn't have relay namespace provisioned appropriately",
-                            Some(ctx.traceId)
-        )
-      )
+      // TODO/PR question: do we still need this RelayNamespace for GCP?
```

Comment:
we don't have RelayNamespace for GCP...r u wondering about Azure?

Reply:
You're right, I got confused- removing this!